plugin/pkg/scheduler/factory/plugins.go

```diff
@@ -61,6 +61,11 @@ const (
 	DefaultProvider = "DefaultProvider"
 )
 
+var (
+	AlgorithmProviders = ""
+	providerCounter    = 1
```

Comment:
Mutable global state! @lavalamp just did a PR to remove all this from the scheduler. See #5351